src/py42/services/users.py

```diff
@@ -11,6 +11,8 @@
 class UserService(BaseService):
     """A service for interacting with Code42 user APIs. Use the UserService to create and retrieve
     users. You can also use it to block and deactivate users.
+
+    `REST Documentation <https://developer.code42.com/api/#tag/User>`__
```

Comment:
I wonder if we should wait to change this one since the gateway has a different meaning for `userId` than what py42 is currently using?

Reply:
Removed. Thanks.